### ISSUE TITLE
How to change faker locale?

### ISSUE THREAD

**Issue by samijnih** [DekodeInteraktiv/anonymize-mysqldump#32]
Hello,

I did some go few years ago but I'm kinda lost right now. I want to use the french locale for faking data.

I don't really understand how I can achieve that?

Thanks

**Comment by PeterBooker** [comment]
Hi @samijnih 

This is a great idea/improvement which was not possible. I have just released version [0.8.0](https://github.com/DekodeInteraktiv/anonymize-mysqldump/releases/tag/0.8.0) which contains the ability to set the locale for the faker package. You can use it by setting the flag `locale` like this:

```sh
cat database.sql | anonymize-mysqldump --config config.json --locale nb_no > anonymized.sql
```

The available locales you can set are [here](https://github.com/DekodeInteraktiv/anonymize-mysqldump/blob/main/internal/anonymize/anonymize.go#L68-L92).

I will close this as solved, but if you have any more questions or any issues using this feature just let us know.

**Comment by samijnih** [comment]
Wow, thank you!
